components/release/k-components/manager-resources-patch/manager_resources_patch.yaml

```diff
@@ -11,7 +11,7 @@ spec:
         resources:
           limits:
             cpu: 500m
-            memory: 4096Mi
+            memory: 8192Mi
```

Comment:
I suggest to set memory request=limits. The reason is that the pod will be allowed to go higher than requests but will be considered in eviction process if node is under pressure. To really reduce the number of restart, set the request as well to 8192Mi

Reply:
thanks for the hint. Changed.

Reply:
Although thinking now, would this not be risky in case the cluster can not give me the requests and the Pod would not spawn?